GVFS/GVFS.UnitTests/Platform.Mac/MacFileSystemVirtualizerTests.cs

```diff
@@ -189,7 +279,7 @@ public void OnEnumerateDirectoryReturnsSuccessWhenResultsNotInMemory()
                 backgroundFileSystemTaskRunner: backgroundTaskRunner,
                 fileSystemVirtualizer: virtualizer))
             {
-                gitIndexProjection.MockFileModes.TryAdd("test" + Path.DirectorySeparatorChar + "test.txt", FileMode644);
+                gitIndexProjection.MockFileTypesAndModes.TryAdd("test" + Path.DirectorySeparatorChar + "test.txt", Regular644FileTypeAndMode);

```

Comment:
This path is used a bunch but in some places it is using Path.Combine and others `"test" + Path.DirectorySeparatorChar + "test.txt"`.  Do you know why?  Is this an inconsistency that we should clean up?  Should there be a const for this path?